BSP/Simulator/Src/BSP_ADC.c

```diff
@@ -16,7 +16,7 @@ static bool initialized = false;
  */
 void BSP_ADC_Init(void) {
     initialized = true;
-    Simulator_Log(LOG, "ADC initialized\n");
+    Simulator_Log_Location(LOG, "ADC initialized\n");
```

Comment:
Some of these are different locations. For example, ADC is LOG and CAN is LOG_INFO. Is there a reason for the change? If so is it documented somewhere?